cnames_active.js

```diff
@@ -2254,7 +2255,7 @@ var cnames_active = {
   "wcfactory": "elmsln.github.io/wcfactory.js.org",
   "wdd": "wangduanduan.github.io",
   "weaver": "maxkfranz.github.io/weaver", // noCF? (don´t add this in a new PR)
-  "weather":"phanendraguptha.github.io/weather"
+  "weather":"phanendraguptha.github.io/weather",
```

Comment:
```suggestion
  "weather":"phanendraguptha.github.io/weather"
```
Please revert this change, your PR should only touch the single line you are using for your request :)

Reply:
OK!